internal/api/connectors/cloudConnector.go

```diff
@@ -40,7 +40,7 @@ type CloudConnectorClient interface {
 
 	GetConnectionStatus(
 		ctx context.Context,
-		account string,
```

Comment:
Not needed. Logging this is nice but we should not be doing translation just for the sake of logging the account i.e. remove translation from recipientStatus controller.